api-gateway/src/main/resources/application-dev.yml

```
@@ -15,11 +15,16 @@ spring:
     oauth2:
       client:
         provider:
-          # example: https://github.com/wearearima/spring-boot-dex/blob/master/src/main/resources/application.properties
+          huawei:
+            authorization-uri: https://oauth-login.cloud.huawei.com/oauth2/v3/authorize
+            token-uri: https://oauth-login.cloud.huawei.com/oauth2/v3/token
+            user-info-uri: https://oauth-login.cloud.huawei.com/oauth2/v3/userinfo
+            user-name-attribute: name
           gitee:
             authorization-uri: https://gitee.com/oauth/authorize
             token-uri: https://gitee.com/oauth/token
             user-info-uri: https://gitee.com/api/v5/user
+          # example: https://github.com/wearearima/spring-boot-dex/blob/master/src/main/resources/application.properties
```

if its necessary info, probably better to link the concrete commit hash, i.e permalink

```suggestion
          # example: https://github.com/wearearima/spring-boot-dex/blob/ad7dd7f501f8fc3946fc87184b4b8b705713ead9/src/main/resources/application.properties
```